### ISSUE TITLE
[7] Skip link not visible when it has keyboard focus

### ISSUE THREAD

**Issue by gordcorp** [AusDTO/dta-website#280]

**Comment by bjgordon** [comment]
we'll have to re-test and investigate - will need a device to reproduce.
